client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
+			wasSubmitted: false,
+			surveyName: null,
+			recommendationValue: null,
+		};
+		this.handleDismissed = this.handleDismissed.bind( this );
+	}
+
+	handleDismissed( event ) {
+		this.setState( {
+			wasDismissed: true,
+			wasSubmitted: event.wasSubmitted,
+			surveyName: event.surveyName,
+			recommendationValue: event.recommendationValue
+		} );
+	}
+
+	render() {
+		// Since this component is still in-progress and unstyled, we will hide it from
```

Comment:
I went back and forth between doing the check here and in `blocks.jsx` as you suggest. I put it here to avoid 'cluttering' `blocks.jsx`, but since you suggested putting the check there, I think I will move it there -- it certainly is a bit clearer.